src/config.rs

```diff
@@ -675,20 +677,22 @@ mod tests {
     }
 
     #[test]
-    fn table_get_styles_bold_italic_underline_green_dimmy_silly_caps() {
-        let config = Value::from("bOlD ItAlIc uNdErLiNe GrEeN diMMeD");
+    fn table_get_styles_bold_italic_underline_green_dimmed_reverse_silly_caps() {
```

Comment:
Yeah, I know that this covers both regular and inverted styles but I'd rather separate those by c/p. Having dedicated tests seems easier on the eyes IMHO.

Reply:
Sorry, could you clarify what you mean by "c/p"? Do mean splitting the function to have one test case for each keyword?

Reply:
My bad, I should've used clearer language. Anyway, I meant copy-pasting the existing test and extending a copy rather than the original, so it results in having both existing `table_get_styles_bold_italic_underline_green_dimmy_silly_caps` and a newly added `table_get_styles_bold_italic_underline_green_dimmy_reverse_silly_caps`.